packages/python/src/browser/python-frontend-module.ts

```diff
@@ -17,8 +17,12 @@
 import { ContainerModule } from "inversify";
 import { LanguageClientContribution } from "@theia/languages/lib/browser";
 import { PythonClientContribution } from "./python-client-contribution";
+import { PythonGrammarContribution } from "./python-grammar-contribution";
+import { LanguageGrammarDefinitionContribution } from "@theia/monaco/lib/browser/textmate";
```

Comment:
dependency to `@theia/monaco` is missing in `package.json`

Reply:
`files.data` property is missing, too

Reply:
done